homeassistant/components/ring/camera.py

```diff
@@ -48,7 +48,7 @@ async def async_setup_entry(
     async_add_entities(cams)
 
 
-class RingCam(RingEntity, Camera):
+class RingCam(RingEntity[RingDoorBell], Camera):
```

Comment:
You you remove `_device: RingDoorBell` here as well?

Reply:
That's done and in `sensor.py` as well.